requirements/pytorch/extra.txt

```diff
@@ -6,5 +6,5 @@ matplotlib>3.1, <3.6.2
 omegaconf>=2.0.5, <2.4.0
 hydra-core>=1.0.5, <1.4.0
 jsonargparse[signatures]>=4.18.0, <4.19.0
-rich>=10.14.0, !=10.15.0.a, <13.0.0
```

Comment:
Required to get the `oldest` matrix test to pass (our script changes `>=` to `==` which causes a dependency conflict -> `==10.14.0` v `>12.3.0`)

Reply:
Thinking we can bump the version also here to be consistent..
cc: @awaelchli @carmocca

Reply:
I added a note in the changelogs.